python/fds.protobuf.stach.v2/setup.py

```diff
@@ -7,6 +7,6 @@
       author='analytics-reporting',
       author_email='analytics.api.support@factset.com',
       license='Apache License 2.0',
-      install_requires=["protobuf"],
+      install_requires=["protobuf>=3.12.0,<4.0.0"],
```

Comment:
Can you please increment the patch version to `1.0.2` for this package?

Reply:
updated